cldfbench_steinertthrelkeldmodals.py

```diff
@@ -19,28 +19,52 @@ def cmd_makecldf(self, args):
         # NOTE: should the column names use `term_uri` and, if so, how?
         args.writer.cldf.add_table(
             "unit-parameters.csv",
-            "ID",
+            {
+                "name": "ID",
+                "propertyUrl": "http://cldf.clld.org/v1.0/terms.rdf#id",
+            },
             "Name",
             "Description",
             "force",
             "flavor",
         )
         args.writer.cldf.add_table(
             "unit-values.csv",
-            "ID",
-            "Language_ID",
-            "Parameter_ID",
-            "Value",
+            {
+                "name": "ID",
+                "propertyUrl": "http://cldf.clld.org/v1.0/terms.rdf#id",
```

Comment:
The `add_*` methods to change the schema accept CSVW column descriptions as `dict` as it would be parsed from JSON. So no need to use `term_uri` here, just specify the URI as `str` for `propertyUrl`.